android/build.gradle

```diff
@@ -30,6 +30,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.19.+'
+    compile 'com.facebook.react:react-native:0.32.+'
```

Comment:
Hi ,

Can you make this :  ```compile 'com.facebook.react:react-native:+``` . This will make it to work with even 33 ( basically version agnostic ) , currently this doesn't work with 33 , I m getting the following error with RN 0.33 while compiling my app
```
A problem occurred configuring project ':app'.
> A problem occurred configuring project ':react-native-android-tablayout'.
  > Could not resolve all dependencies for configuration ':react-native-android-tablayout:_debugCompile'.
     > Could not find any version that matches com.facebook.react:react-native:0.32.+.
       Versions that do not match:
           0.20.1
           0.20.0
           0.19.1
           0.19.0
           0.18.0
           + 13 more
```